CONTRIBUTING.md

```diff
@@ -86,13 +86,16 @@ You can access the app [localhost:3000](http://localhost:3000). Whatever you cha
 
 ### Commits
 * Commit title should be short and descriptive
-  * title (or summary line) is the first line of the commit message
-  * that says what the commit is doing
-  * in no more than 50 characters
-  * starting with a word like 'Fix' or 'Add' or 'Change'
+  * Title (or summary line) is the first line of the commit message
+  * That says what the commit is doing in imperative mood (Fix foobar)
+  * In no more than 50 characters
+  * Starting with a word like 'Fix' or 'Add' or 'Change'
```

Comment:
Can we also add _Enable_ here? So:

_Starting with a word like 'Fix' or 'Add', 'Change' or 'Enable'_